packages/replayio/src/utils/launch-darkly/initLaunchDarklyFromAccessToken.ts

```diff
@@ -24,7 +25,7 @@ export async function initLaunchDarklyFromAccessToken(accessToken: string) {
       await identifyUserProfile(id);
     }
   } catch (error) {
-    console.error(error);
+    debug("Failed to initialize LaunchDarkly profile: %o", error);
```

Comment:
`debug` gets swallowed up by default (basically for any public user) which is why I was using `console.error` in a few places instead. Why this change?

Reply:
Launchdarkly is not a critical piece of the CLI - so even if it fails we should be able to cope with it.

At the moment the user is greeted with this:
<img width="1097" alt="Errors printed in the stdout" src="https://github.com/replayio/replay-cli/assets/9800850/03bfcfda-5869-447c-96c1-c57214565b23">

Reply:
Yeah that error message is kind of noisy and unhelpful. Okay this change makes sense.